build.gradle

```diff
@@ -32,11 +32,14 @@ java {
 }
 
 dependencies {
+
     compileOnly  "org.embulk:embulk-spi:0.11"
     implementation "org.embulk:embulk-util-config:0.5.0"
     implementation "org.embulk:embulk-util-file:0.2.0"
 
     testImplementation "junit:junit:4.13.2"
+    testImplementation "org.embulk:embulk-deps:0.11.3"
```

Comment:
We may want to have `embulk-core` as well, explicitly.

Reply:
`org.embulk:embulk-core` added.